tests/features/consumer/searchDocumentReference-success.feature

```diff
@@ -19,6 +19,7 @@ Feature: Consumer - searchDocumentReference - Success Scenarios
       | subject   | 9278693472 |
     Then the response status code is 200
     And the response is a searchset Bundle
+    And the Bundle has a self link matching consumer/FHIR/R4/DocumentReference?subject:identifier=https://fhir.nhs.uk/Id/nhs-number|9278693472"
```

Comment:
is it worth commenting this out until after the integration test ticket is done? that way we can merge this ticket as-is and pull the integration tests in once they're ready?

Reply:
nice fix, even better :clap: